### DISCUSSION TITLE
string has no attribute train

### DISCUSSION THREAD

**Discussion by Valkyrri025** [xN1ckuz/Crosswalks-Detection-using-YOLO#4] [General]
Dear developer, thank you for your work. When I run your model in Google colab, error appears in Training section that "model" has been defined to be a string and "str" has no attribute "train". Is this error just for me and how do I solve this problem? Thank you!

**Comment by xN1ckuz** [comment]
> Dear developer, thank you for your work. When I run your model in Google colab, error appears in Training section that "model" has been defined to be a string and "str" has no attribute "train". Is this error just for me and how do I solve this problem? Thank you!

Ok i looked at the code, seems like there is a problem loading the model. I submitted a Bug Report directly to UItralytics, i will update you there for any changes. Sorry for the incovenience, the code was working fine months ago.

**Reply by Valkyrri025** [reply]
No worries. Thank you!

**Reply by xN1ckuz** [reply]
I looked at it a little bit. It's fixed now, try it and let me know.
